gtd-timer/Exceptions/UserExistException.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Runtime.Serialization;
+
+namespace gtdtimer.Exceptions
+{
+    public class UserExistException : Exception
```

Comment:
"UserAlreadyExistsException" would be better in my opinion

Reply:
Done